src/test/java/core/basesyntax/service/impl/ReaderServiceTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReaderServiceTest {
+    private static final String EMPTY_FILE_NAME = "src/test/resources/empty.csv";
+    private static final String NOT_EXISTED_FILE_NAME = "src/test/resources/notExistedFile.csv";
+    private static final String EXISTED_FILE_NAME = "src/test/resources/input.csv";
+    private static ReaderService readerService;
+
+    @BeforeClass
+    public static void readerServiceInitialization() {
+        readerService = new ReaderServiceImpl();
+    }
+
+    @Test
```

Comment:
```suggestion
    @Test(expected = RuntimeException.class)
```
You can use this construction